doc-content/enterprise-only/openshift/rhdm73-optaweb-employee-rostering-ref.adoc

```diff
@@ -0,0 +1,282 @@
+|`OPTAWEB_HTTPS_KEYSTORE` | `HTTPS_KEYSTORE` | The name of the keystore file within the secret | keystore.jks | False
+|`OPTAWEB_HTTPS_NAME` | `HTTPS_NAME` | The name associated with the server certificate | jboss | False
+|`OPTAWEB_HTTPS_PASSWORD` | `HTTPS_PASSWORD` | The password for the keystore and certificate | mykeystorepass | False
+|`OPTAWEB_MEMORY_LIMIT` | -- | Optaweb employee rostering Container memory limit | 1Gi | False
+|`OPTAWEB_GENERATOR_ZONE_ID` | `OPTAWEB_GENERATOR_ZONE_ID` | The timezone id used by the OptaWeb Generator (Sets the optaweb.generator.zoneId system property; defaults to local timezone) | -- | False
+|=======================================================================
+
+
+
+== Objects
+
+The CLI supports various object types. A list of these object types as well as their abbreviations
+can be found in the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cli_reference/cli-reference-basic-cli-operations#object-types[Openshift documentation].
+
+
+=== Services
+
+A service is an abstraction which defines a logical set of pods and a policy by which to access them. Refer to the
+https://cloud.google.com/container-engine/docs/services/[container-engine documentation] for more information.
+
+|=============
+|Service        |Port  |Name | Description
+
+.2+| `${APPLICATION_NAME}-optaweb-employee-rostering`
+|8080 | http
+.2+| All the Optaweb employee rostering web server's ports.
+|8443 | https
+.1+| `${APPLICATION_NAME}-optaweb-employee-rostering-ping`
+|8888 | ping
+.1+| The JGroups ping port for clustering.
+.1+| `${APPLICATION_NAME}-postgresql`
+|5432 | --
+.1+| The database server's port.
+|=============
+
+
+
+=== Routes
+
+A route is a way to expose a service by giving it an externally-reachable hostname such as `www.example.com`. A defined route and the endpoints
+identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
+of a route name, service selector, and (optionally) security configuration. Refer to the
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+
+|=============
+| Service    | Security | Hostname
+
+|`${APPLICATION_NAME}-optaweb-employee-rostering-http` | none | `${OPTAWEB_HOSTNAME_HTTP}`
+|`${APPLICATION_NAME}-optaweb-employee-rostering-https` | TLS passthrough | `${OPTAWEB_HOSTNAME_HTTPS}`
+|=============
+
+
+
+
+=== Deployment Configurations
+
+A deployment in OpenShift is a replication controller based on a user defined template called a deployment configuration. Deployments are created manually or in response to triggered events.
+Refer to the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/deployments#dev-guide-how-deployments-work[Openshift documentation] for more information.
```

Comment:
For more information about (x), see (y).

Reply:
as above for post 7.3